README.rst

```diff
@@ -37,10 +37,10 @@ Make sure you have docker_ installed to try this quickstart.
     $> cd catchpy
 
     # start docker services
-    $> docker compose up
-    $> docker compose exec web python manage.py migrate
-    $> docker compose exec web python manage.py createsuperuser
-    $> open http://localhost:8000/static/anno/index.html
+    $> docker-compose up
+    $> docker-compose exec web python manage.py migrate
+    $> docker-compose exec web python manage.py createsuperuser
+    $> open http://localhost:9000/static/anno/index.html
```

Comment:
Looks like the legacy `docker-compose` command got put back in. I think Docker will soon stop shipping with the legacy command. It is not just syntax, `docker-compose` is a different CLI.

Reply:
I've updated the docker commands.